soccer/planning/MotionConstraints.hpp

```diff
@@ -57,7 +31,8 @@ struct MotionConstraints {
     static void createConfiguration(Configuration* cfg);
     static ConfigDouble* _max_acceleration;
     static ConfigDouble* _max_speed;
-    static ConfigDouble* _max_angle_speed;
+    static ConfigDouble* _max_rotation_speed;
```

Comment:
It's in the todo
On Sep 10, 2015 2:54 AM, "Justin Buchanan" <notifications@github.com> wrote:

> In soccer/planning/MotionConstraints.hpp
> <https://github.com/RoboJackets/robocup-software/pull/444#discussion_r39129585>
> :
>
> > @@ -57,7 +31,8 @@ struct MotionConstraints {
> >      static void createConfiguration(Configuration* cfg);
> >      static ConfigDouble* _max_acceleration;
> >      static ConfigDouble* _max_speed;
> > -    static ConfigDouble* _max_angle_speed;
> > +    static ConfigDouble* _max_rotation_speed;
>
> Let's move these to the RotationConstraints class.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RoboJackets/robocup-software/pull/444/files#r39129585>
> .
>